modules/tensor_mechanics/include/materials/ComputeCappedDruckerPragerStress.h

```diff
@@ -7,7 +7,7 @@
 #ifndef COMPUTECAPPEDDRUCKERPRAGERSTRESS_H
```

Comment:
Now that this is an 'update' model, its name should be named to match our convention: `CappedDruckerPragerStressUpdate.h`